docs/tableOfContents.js

```diff
@@ -232,6 +232,11 @@ export default [
         isCode: true,
         keywords: buttonRelatedKeywords,
       }),
+      new Page({
+        path: '/kimg',
+        title: 'KImg',
+        isCode: true,
```

Comment:
It would be nice to add the `keywords` property to allow for better searching within the documentation. For example, currently when you try to search an image component using the "image" keyword, the search would yield no results. Please see usages within this file for implementation details

Reply:
I think the changes made should suffice. Thanks @thanksameeelian. Non-blocking but would be good to follow existing patterns i.e. pass variable to keywords as opposed to the actual array object

Reply:
thanks! i checked out that pattern when i was adding the keywords here, but only saw the pattern of using a variable when multiple items had the same keywords, so the variable was being used in at least two places. 

i don't think that pattern is applicable here, as `KImg` is the only item using those specific keywords. i am following the pattern i saw for each item that had a unique list of variables, as in those cases the array itself was passed. you can see this patten in `KSelect` and `KSwitch` in the table of contents. i think this makes sense because it avoids unnecessary abstraction. 

but if another item were to share the same keywords at kimg, i would totally agree that a variable would be the right call!